packages/gafl-webapp-service/src/pages/order-complete/licence-details/__tests__/route.spec.js

```diff
@@ -22,73 +26,188 @@ jest.mock('../../../../constants', () => ({
+      licenceTypeDisplay.mockReturnValue(licenceType)
+
+      const licenceLength = Symbol('8 days')
+      licenceTypeAndLengthDisplay.mockReturnValue(licenceLength)
+
+      const startTime = Symbol('9:32am on 23 June 2021')
+      displayStartTime.mockReturnValue(startTime)
+
+      const endTime = Symbol('9:32am on 23 June 2022')
+      displayEndTime.mockReturnValue(endTime)
+
+      const disabled = Symbol('disabled')
+      concessionHelper.hasDisabled.mockReturnValue(disabled)
+
+      const concession = Symbol('concession')
+      concessionHelper.getAgeConcession.mockReturnValue(concession)
+
+      it('referenceNumber', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].referenceNumber).toBe('123456789')
+      })
+
+      it('licenceHolder', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].licenceHolder).toBe('Turanga Leela')
+      })
+
+      it('obfuscatedDob', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].obfuscatedDob).toBe('obfuscatedDob')
+      })
+
+      it('type', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].type).toBe(licenceType)
+      })
+
+      it('length', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].length).toBe(licenceLength)
+      })
+
+      it('start', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].start).toBe(startTime)
+      })
+
+      it('end', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].end).toBe(endTime)
+      })
+
+      it('price', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].price).toBe(permission.permit.cost)
+      })
+
+      it('disabled', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].disabled).toBe(disabled)
+      })
+
+      it('ageConcession', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].ageConcession).toBe(concession)
+      })
+
+      it('index', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data.licences[0].index).toBe(0)
+      })
+
+      it('returns expected data', async () => {
+        const data = await getData(getMockRequest(status, transaction))
+        expect(data).toMatchSnapshot()
+      })
+    })
+
+    describe('throws a Boom forbidden error', () => {
+      it('if status agreed flag is not set', async () => {
+        const status = () => ({})
```

Comment:
As a continuation of my point about `status` and `transaction` above - you never actually set a different value for transaction. So there's no need to have it as a parameter in the `getMockRequest` function